Source/diablo.cpp

```diff
@@ -812,6 +823,10 @@ void RunGameLoop(interface_mode uMsg)
 
 void DiabloParseFlags(int argc, char **argv)

```

Comment:
Line 819: `printInConsole(" %-20s %-30s\n", "+<internal command>", "Pass commands to the engine");`

Reply:
Added 🙂